stdlib/asm/crypto/dsa/ecdsa/secp256k1.masm

```diff
@@ -22,6 +24,69 @@ const.SIG_LEN_FELTS=17   # 66.div_ceil(4)
 # PROCEDURES
 # ================================================================================================
 
+#! Prepares calldata for deferred secp256k1 verification.
+#!
+#! This wrapper mirrors the interface of `rpo_falcon512::verify`: the commitment to the public key
+#! and the message arrive on the operand stack, while the concrete public key and signature bytes
+#! are streamed from the advice stack. The routine stages the calldata in local memory before
+#! invoking the precompile.
+#!
+#! Inputs:
+#!   Operand stack: [PK_RPO, MSG_WORD, ...]
```

Comment:
nit: I would maybe use `PK_COMMITMENT` instead of `PK_RPO`. In `miden-base` we have `PublicKeyCommitment` type which corresponds to this.